apisix/plugins/ext-plugin/init.lua

```diff
@@ -279,7 +279,12 @@ local function handle_extra_info(ctx, input)
         local reqbody_req = extra_info_reqbody.New()
         reqbody_req:Init(info.bytes, info.pos)
 
-        res = core.request.get_body()
+        local err
+        res, err = core.request.get_body()
+        if not res then
```

Comment:
We need to check the err instead, as the body can be nil when there is no body.

Reply:
update